lib/jellyfish/room_service.ex

```diff
@@ -57,9 +57,12 @@ defmodule Jellyfish.RoomService do
   @spec create_room(Room.max_peers(), String.t()) ::
           {:ok, Room.t(), String.t()} | {:error, :invalid_max_peers | :invalid_video_codec}
   def create_room(max_peers, video_codec) do
-    :ok = Phoenix.PubSub.broadcast(Jellyfish.PubSub, "jellyfishes", {:get_resource_usage, self()})
+    {node_resources, failed_nodes} =
+      :rpc.multicall(Jellyfish.RoomService, :get_resource_usage, [])
```

Comment:
In theory, node_resources might contain tuples `{:badrpc, Reason}`. Not sure if we should filter and log them?

Reply:
Yup, I found out about that in #59. After fixing it there, I will also add this fix here.

Reply:
kk, we can merge this without waiting for #59 and fix this in #59